src/classic/clvm_tools/stages/stage_2/optimize.rs

```diff
@@ -230,15 +221,15 @@ fn path_from_args<'a>(
     }
 }
 
-fn sub_args<'a>(
-    allocator: &'a mut Allocator,
+pub fn sub_args(
```

Comment:
having to make this `pub` is the price of moving out the tests. I think having tests in the same file is nice, but it's not a big deal